frontend/src/static/js/utils/constants.ts

```diff
@@ -87,6 +87,34 @@ export const TOP_HTML_INTEROP_ISSUES: string[] = [
   'web-bluetooth',
 ];
 
+export const INTEROP_FEATURES: string[] = [
```

Comment:
I know this is still in draft, but could you add a comment that links to the source for this? In case we need to update for future years

Reply:
Great idea, I've added it. 🙂

The file that contains this mapping [hasn't landed yet](https://github.com/web-platform-tests/interop/pull/930), so we can keep this PR as a draft until it's ready.